Update backups-tool layout to WordPress layout

nasusa opened the pull request:
- Make backups-tools content responsive

- Fix code indentation

- Make backups.js file and move JavaScript there

- Make function register_scripts to backups.php

- Make backups.css-file and move css there

I updated backups-tool layout to wordpress layout.

When content are on postboxes it is easier to find what you are looking for. Different tools are more separated and when you go first time to look at the tool, you will see immediately what data and features tool contains.

![backups_tool](https://user-images.githubusercontent.com/15683849/40534960-5b2f6ea2-6010-11e8-8de0-3073bf351207.png)

ottok commented:
UI looks nice!

Git-tip: if you add .patch after the Github commit url, you can see the commit in patch format as if it was submitted by email: https://github.com/Seravo/seravo-plugin/commit/0836725f2a9bd856844fa3719c619cea4c231c64.patch

Your author email is incorrect:
```
From 0836725f2a9bd856844fa3719c619cea4c231c64 Mon Sep 17 00:00:00 2001
From: Susanna Lepola <suskaari@hotmail.com>
Date: Thu, 24 May 2018 09:28:22 +0300
Subject: [PATCH] Update backups-tool layout to wordpress layout

Make backups-tools content responsive

Fix code indentation

Make backups.js file and move JavaScript there

Make function register_scripts to backups.php

Make backups.css-file and move css there
```


Please run on your computer
```
git commit --amend --author "Susanna Lepola <susanna@seravo.fi>"
git push -f origin backups-tool
# and for future
git config --local user.email susanna@seravo.fi
```

See `git config --help` so see what this means.

l3ku commented:
Couldn't get the postbox close/open or "Create a new backup" buttons to work (emptied cache), please check that the JavaScript is loaded properly.

nasusa commented:
I fixed that problem. Same problem was in database-tool, I fixed it also.

"Use ready() to make a function available after the document is loaded:"

```
jQuery(document).ready(function($)
```
